### PR TITLE
Added dataset clickbait_news_bg

### PR DESCRIPTION
**Pull request by tsvm**

### CODE REVIEW

File: datasets/clickbait_news_bg/clickbait_news_bg.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+class ClickbaitNewsBGDataset(datasets.GeneratorBasedBuilder):
+    VERSION = datasets.Version("1.1.0")
+    DEFAULT_CONFIG_NAME = "default"
+
+    def _info(self):
+        if self.config.name == "default":
+            features = datasets.Features(
+                {
+                    "fake_news_score": datasets.Value("int8"),
+                    "click_bait_score": datasets.Value("int8"),
```
Comment (SBrandeis) [inline datasets/clickbait_news_bg/clickbait_news_bg.py:65]: What are the values this feature can take ?
It feels like `click_bait_score` and `fake_news_score` could be `ClassLabel` features

**Review by SBrandeis — COMMENTED** [review]
Great dataset, thanks 🙂 
I have a few remarks ⬇️

File: datasets/clickbait_news_bg/clickbait_news_bg.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {
+    "default_train": "https://gitlab.com/datasciencesociety/case_fake_news/-/raw/master/data/FN_Training_Set.xlsx",
+    "default_validation": "https://gitlab.com/datasciencesociety/case_fake_news/-/raw/master/data/FN_Validation_Set.xlsx",
+}
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class ClickbaitNewsBGDataset(datasets.GeneratorBasedBuilder):
```
[the remark this answers is not in the document]
Comment (lhoestq) [reply datasets/clickbait_news_bg/clickbait_news_bg.py:56]: indeed the name should be `ClickbaitNewsBG` to match the dataset script name

File: datasets/clickbait_news_bg/clickbait_news_bg.py (hunk 1 above)
Comment (lhoestq) [reply datasets/clickbait_news_bg/clickbait_news_bg.py:65]: It looks like the labels take only the value 1 (negative prediction) and 3 (positive prediction).
You can use ClassLabel this way for example:
```
"click_bait_score": ClassLabel(names=["normal", "click bait"])
"fake_news_score": ClassLabel(names=["legitimate", "fake"])
```
(label names come from [here](https://gitlab.com/datasciencesociety/case_fake_news/-/blob/master/case_study/Hackaton_Task.pdf))

You will also have to convert all the "1" and the "3" accordingly when you yield the examples in _generate_examples:
- for click_bait_score:
  - 1 -> "normal"
  - 3 -> "click bait"
- for fake_news_score:
  - 1 -> "legitimate"
  - 3 -> "fake"

**Review by lhoestq — COMMENTED** [review]
This is really cool thanks !

I left a few comments.

After the changes of the class name and the feature type, you'll need to regenerate the dataset_infos.json file, and we'll be done :)

**Comment by lhoestq** [comment]
Looks like this PR includes changes about many other files than the ones for clickbait_news_bg

Can you create another branch and another PR please ?

**Comment by tsvm** [comment]
> PR

I created a new branch with the dataset code and submitted a new PR for it: https://github.com/huggingface/datasets/pull/1568

**Pull request closed without merging**